numpy/core/src/umath/ufunc_object.c

```diff
@@ -2124,18 +2124,27 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
 
     if (axes) {
         /*
-         * possibly remap axes.
+         * possibly remap axes, using newly allocated memory.
          */
+        remap_axis = (npy_intp *)PyArray_malloc(NPY_SIZEOF_INTP * nop);
+        remap_axis_memory = (int *)PyArray_malloc(sizeof(int) * nop *
```

Comment:
Something is wrong here - these are declared as `int**` and `int*`, respectively., so the casts don't match.

Reply:
Yes, silly. I'm not much good with C pointer magic... (which perhaps is another reason not to change it...).